automation_testing/pages/FormFields/FormFieldTextPage.ts

```diff
@@ -0,0 +1,65 @@
+import { Locator, Page } from "@playwright/test";
+import { BasePage } from "../BasePage";
+import { generateRandomId } from "../../utils/helpers/helper";
+
+export class FormFieldTextPage extends BasePage {
+
+	readonly page_path = "formfields-formfieldtext--kitchen-sink";
+
+	readonly page: Page;
+	readonly regularTextField: Locator;
+	readonly numberTextField: Locator;
+	readonly multilineTextField: Locator;
+	readonly fieldMaxCharTextField: Locator;
+	readonly maxCharCounter: Locator;
+	readonly fieldWithIconDiv: Locator;
+	readonly fieldWithIconIcon: Locator;
+	readonly fieldWithIconTextField: Locator;
+	readonly disabledTextField: Locator;
+	readonly xsSizeTextField: Locator;
+	readonly smSizeTextField: Locator;
+	readonly mdSizeTextField: Locator;
+	readonly lgSizeTextField: Locator;
+	
+	constructor(page: Page) {
+		super(page);
+		this.page = page;
+		this.regularTextField = page.locator("#regular");
+		this.numberTextField = page.locator("#number");
+		this.multilineTextField = page.locator("#multiline");
+		this.fieldMaxCharTextField = page.locator("#withMaxChar");
+		this.maxCharCounter = page.locator("//*[@id='root']/div/form/div[2]/div[4]/div/div/div/div/div[1]/div[1]/div[2]");
+		this.fieldWithIconDiv = page.locator("//*[@id='root']/div/form/div[2]/div[5]/div/div/div/div/div[1]/div[2]/div");
+		this.fieldWithIconIcon = page.locator("[data-testid='AccountCircleIcon']");
+		this.fieldWithIconTextField = page.locator("#withIcon");
+		this.disabledTextField = page.locator("#disabled");
+		this.xsSizeTextField = page.locator("#xsSize");
+		this.smSizeTextField = page.locator("#smSize");
+		this.mdSizeTextField = page.locator("#mdSize");
+		this.lgSizeTextField = page.locator("#lgSize");
+	}
+
+	async visitPage(): Promise<void> {
+		await this.visit(this.page_path, this.title);
+	}
+
+	async getAutogeneratedText(length: number): Promise<string> {
+		return generateRandomId(length);
+	}
+
+	async getLimitOfMaxChar(): Promise<number> {
```

Comment:
Typescript is complaining that the value of `this.maxCharCounter.textContent()` could be possibly null. If you are 100% sure that this value will never be null you can use the non-null assertion operator to remove this warning. https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-
If there is a chance that this value is equal to null, you add a check for this case.

```
(property) FormFieldTextPage.maxCharCounter: Locator
Object is possibly 'null'.ts(2531)
```

![image](https://user-images.githubusercontent.com/87880265/180000970-65d3de7e-2839-4194-a3f3-293bdea09c35.png)

### Solution
![image](https://user-images.githubusercontent.com/87880265/180001925-291645f9-2a1b-4ab7-a45f-0bcdf535db29.png)

Reply:
Yes! That will do the work.